etc/profile-a-l/liferea.profile

```diff
@@ -49,7 +50,10 @@ shell none
 tracelog
 
 disable-mnt
+#private-bin liferea
+private-cache
```

Comment:
Liferea drops several files into ${HOME}/.cache/liferea. Enabling private-cache will discard those on sandbox shutdown. Is this intentional here? Amongst other things this implies feed favicons would have to be redownloaded on each invocation. Personally I don't see much harm in dropping private-cache in this particular case. Just wondering what your motivation is to add it to the profile...

Reply:
Right, I was testing with a small number of feeds synced to my ttrss instance and didn't notice the difference. Tried on my main profile >200 feeds it it is an issue with slow networks and feed numbers.